microsoft/testsuites/network/sriov.py

```diff
@@ -542,6 +544,158 @@ def sriov_ethtool_offload_setting(self, environment: Environment) -> None:
+            # 6. Get final interrupts sum per cpu on client node.
+            final_pci_interrupts_by_cpus = Interrupt.get_sum_of_interrupt_data_by_cpus(
+                client_node, client_nic_info.pci_slot
+            )
+            for cpu, init_interrupts_value in initial_pci_interrupts_by_cpus.items():
+                final_interrupts_value = final_pci_interrupts_by_cpus[cpu]
+                # 7. Compare interrupts changes, expected to see interrupts increased.
+                assert_that(final_interrupts_value).described_as(
+                    f"CPU {cpu} didn't have an increased interrupts count"
+                    " after iperf3 run!"
+                ).is_greater_than(init_interrupts_value)
+
+    @TestCaseMetadata(
+        description="""
+        This case is to check whether the network connectivity is lost after running
+         iperf3 for 30 mins.
+
+        Steps,
+        1. Start iperf3 on server node.
+        2. Start iperf3 for 30 minutes on client node.
+        3. Do VF connection test.
+        """,
+        priority=3,
+        timeout=18000,
+        requirement=node_requirement(
+            node=schema.NodeSpace(
+                node_count=2,
+                network_interface=schema.NetworkInterfaceOptionSettings(
+                    nic_count=search_space.IntRange(min=1, max=8),
+                    data_path=schema.NetworkDataPath.Sriov,
+                ),
+            )
+        ),
+    )
+    def sriov_iperf_stress(self, environment: Environment) -> None:
+        server_node = cast(RemoteNode, environment.nodes[0])
+        client_node = cast(RemoteNode, environment.nodes[1])
+        vm_nics = self._initialize_nic_info(environment)
+        client_iperf3_log = "iperfResults.log"
+        server_iperf3 = server_node.tools[Iperf3]
+        client_iperf3 = client_node.tools[Iperf3]
+        # 1. Start iperf3 on server node.
+        server_iperf3.run_as_server()
+
+        # 2. Start iperf3 for 30 minutes on client node.
+        for _, client_nic_info in vm_nics[client_node.name].items():
+            matched_server_nic_info: NicInfo
+            for _, server_nic_info in vm_nics[server_node.name].items():
+                if (
+                    server_nic_info.ip_addr.rsplit(".", maxsplit=1)[0]
+                    == client_nic_info.ip_addr.rsplit(".", maxsplit=1)[0]
+                ):
+                    matched_server_nic_info = server_nic_info
+                    break
+            assert matched_server_nic_info
+            client_iperf3.run_as_client(
+                server_ip=matched_server_nic_info.ip_addr,
+                log_file=client_iperf3_log,
+                seconds=1800,
+                client_ip=client_nic_info.ip_addr,
+            )
+            client_cat = client_node.tools[Cat]
+            iperf_log = client_cat.read_from_file(
+                client_iperf3_log, sudo=True, force_run=True
+            )
+            assert_that(iperf_log).does_not_contain("error")
```

Comment:
Add described_as to show more practical information.

Reply:
udpated